jsonrs/jsoniter.go

```diff
@@ -0,0 +1,38 @@
+package jsonrs
+
+import (
+	"io"
+
+	jsoniter "github.com/json-iterator/go"
+)
+
+// jsoniterJSON is the JSON implementation of github.com/json-iterator/go.
+type jsoniterJSON struct{}
+
+func (j *jsoniterJSON) Marshal(v any) ([]byte, error) {
+	return jsoniter.ConfigCompatibleWithStandardLibrary.Marshal(v)
```

Comment:
We could use a `var defaultJsoniter = jsoniter.ConfigCompatibleWithStandardLibrary` and then do:

```suggestion
	return defaultJsoniter.Marshal(v)
```

This would help us remove the repetition which is error prone.

Alternatively we could expose a constructor.